src/pages/instances/actions/snapshots/CreateSnapshotForm.tsx

```diff
@@ -126,29 +161,54 @@ const CreateSnapshotForm: FC<Props> = ({ instance, close }) => {
           onBlur={formik.handleBlur}
           value={formik.values.name}
           error={formik.touched.name ? formik.errors.name : null}
-          stacked
           takeFocus
         />
-        <Input
-          id="expiresAt"
-          name="expiresAt"
-          type="datetime-local"
-          label="Expires at"
-          min={getTomorrowMidnight()}
-          onChange={formik.handleChange}
-          onBlur={formik.handleBlur}
-          stacked
-        />
-        <Input
-          id="stateful"
-          name="stateful"
-          type="checkbox"
-          label="Stateful"
-          wrapperClassName="row"
-          help={getStatefulHelp()}
-          disabled={!isStateful || !isRunning}
-          onChange={formik.handleChange}
-          onBlur={formik.handleBlur}
+        <Row className="expiration-wrapper">
+          <Col size={6}>
+            <Input
+              id="expirationDate"
```

Comment:
I wonder if we can make more clear the dates are optional. In our user tests, no one realized that, and everyone started to enter a date. Might be a future task for design, though.

Reply:
Sure, feel free to add a note to discuss it in the next UX-UI sync with @al-bakalova